packages/pigeon/platform_tests/flutter_null_safe_unit_tests/lib/all_datatypes.dart

```diff
@@ -111,17 +111,16 @@ class HostEverything {
         details: error['details'],
       );
     } else {
-      return Everything.decode(replyMap['result']!);
+      return (replyMap['result'] as Everything?)!;
```

Comment:
As a follow-up: it would be helpful if these files were in a folder called "gen" or "generated", or had names including "generated". I didn't realize at first that I was reviewing a generated file because the comment at the top isn't in the review context.